xline/src/storage/kvstore.rs

```diff
@@ -522,3 +595,87 @@ impl KvStoreBackend {
         Self::new_deletion_events(revision, prev_kv)
     }
 }
+
+#[cfg(test)]
+mod test {
+    use std::error::Error;
+
+    use super::*;
+
+    #[tokio::test]
+    async fn test_keys_only() -> Result<(), Box<dyn Error>> {
+        let store = init_store().await?;
+
+        let request = RangeRequest {
+            key: vec![0],
+            range_end: vec![0],
+            keys_only: true,
+            ..Default::default()
+        };
+        let response = store.inner.handle_range_request(&request);
+        assert_eq!(response.kvs.len(), 5);
+        for kv in response.kvs {
+            assert!(kv.value.is_empty());
+        }
+
+        Ok(())
+    }
+
+    #[tokio::test]
+    async fn test_range_empty() -> Result<(), Box<dyn Error>> {
+        let store = init_store().await?;
+
+        let request = RangeRequest {
+            key: "x".into(),
+            range_end: "y".into(),
+            keys_only: true,
+            ..Default::default()
+        };
+        let response = store.inner.handle_range_request(&request);
+        assert_eq!(response.kvs.len(), 0);
+        assert_eq!(response.count, 0);
+
+        Ok(())
+    }
+
+    #[tokio::test]
+    #[allow(clippy::indexing_slicing)] // Checked before use index
+    async fn test_range_filter() -> Result<(), Box<dyn Error>> {
```

Comment:
Please enrich this test case to cover different range request parameters, like different `SortOrder` on different `SortTarget`.

Reply:
added a  new test